tp1/freeway/Assets/Scripts/chicken.cs

```diff
@@ -42,18 +57,30 @@ public class chicken : MonoBehaviour {
 		Debug.Log (col2d.tag);
 		if (col2d.tag.Equals ("LittlePinkCar")) {
 			Debug.Log ("perdí :(");
-			transform.position = initialPosition;
+			if (this.tag.Equals ("Chicken")) {
+				transform.position = initialPosition;
```

Comment:
This is the same as before. We could receive the relative positions in the "matrix" and transform then with the assets